mirrormanager2/app.py

```diff
@@ -1436,3 +1436,39 @@ def statistics(date=None, cat='countries'):
 @APP.route('/maps')
 def maps():
         return flask.render_template('maps.html')
+
+
+@APP.route('/propgation')
+@APP.route('/propgation/<prefix>')
+def propagation(prefix='development'):
+    """ Display propagation statistics. The files displayed
+    are generated by mm2_propagation which generates output files
+    like this <prefix>-repomd-propagation.svg.
+    It also generates these files with that date included. For now
+    only the files without the date are displayed.
+    """
+
+    prefix = os.path.basename(prefix)
+    # Right now propagation statistics are only generated
+    # for the 'development' prefix and 'fxx_updates' prefix.
+    if not prefix.startswith('f') and not prefix.startswith('d'):
+        prefix = 'development'
+
+    prop_base = APP.config['PROPAGATION_BASE']
+    stat_file = os.path.join(prop_base, prefix + "-repomd-propagation.svg")
```

Comment:
What about having this run by ``werkzeug.secure_filename()`` to make sure it's safe?